DTS-STN/passport-status
# Update status codes

## [ADO-1308](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1308)

### Description
This PR updates the status codes and corresponding messages & descriptions based on the table in this PR : https://github.com/DTS-STN/passport-status-api/pull/46

This also makes use the manifest number, which was added to the check status response in #118, to display the user's tracking number if their status is either a 3 or a 4 (issued and shipped via Canada Post or Fedex). There's some additional logic to determine whether the status code is either 3 or 4 since there are some records (at least in the mocks) that have a manifest number even though the user's passport hasn't been shipped yet (e.g. we have a record with the status code 5, which is rejected, that has a manifest number)

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/status`
4. Enter the following mock data:
```
ESRF: A0890196
Given Name: Manon
Surname: Le roux
DoB: 1987-02-24
```
5. Press submit and confirm that the tracking number is shown (and it says it's a Canada Post tracking number)
6. Try with other mock data to test out the other status codes (dm if you don't know where to find this)

### Additional Notes
In `status.tsx` CheckStatusReponse should be CheckStatusRe**s**ponse
